src/helpers/__tests__/enhancedBrowserMetrics.test.browser.js

```diff
@@ -37,7 +37,7 @@ test('enhancedBrowserMetrics', t => {
           w,
           null,
           2
-        )}`
+        ) || ''}`
```

Comment:
This looks to be a change to the core definition for `JSON.stringify`.  I'm guessing this failed CI where we are using `flow-bin@latest`?

```js
// flow v0.89.0's core.js
declare class JSON {
    static parse(text: string, reviver?: (key: any, value: any) => any): any;
    static stringify(
      value: any,
      replacer?: ?((key: string, value: any) => any) | Array<any>,
      space?: string | number
    ): string;
}
```

Was [updated](https://github.com/facebook/flow/blob/0bd10f12a1a17ba157d654116e551be8411dacd0/lib/core.js#L480-L492) to:
```js
declare class JSON {
    static parse(text: string, reviver?: (key: any, value: any) => any): any;
    static stringify(
      value: null | string | number | boolean | {} | $ReadOnlyArray<mixed>,
      replacer?: ?((key: string, value: any) => any) | Array<any>,
      space?: string | number
    ): string;
    static stringify(
      value: mixed,
      replacer?: ?((key: string, value: any) => any) | Array<any>,
      space?: string | number
    ): string | void;
}
```

https://github.com/facebook/flow/blob/0bd10f12a1a17ba157d654116e551be8411dacd0/lib/core.js#L480-L492

Reply:
yes, in monorepo sync